roles/springboot/tasks/springboot.yml

```diff
@@ -0,0 +1,46 @@
+---
+
+- name: "{{ springboot.name }} | Set facts (1)"
+  set_fact:
+    _springapp_service_name: "{{ springboot.name }}"
+    _springapp_type: "{{ springboot.type if springboot.type is defined else 'gui' }}"
+
+- name: "{{ springboot.name }}-{{ _springapp_type }} | Set facts (2)"
+  set_fact:
+    _springapp_min_heapsize: "{{ springboot.min_heapsize if springboot.min_heapsize is defined else springboot_min_heapsize }}"
+    _springapp_max_heapsize: "{{ springboot.max_heapsize if springboot.max_heapsize is defined else springboot_max_heapsize }}"
+    _springapp_random_source: "{{ springboot.random_source if springboot.random_source is defined else springboot_random_source }}"
+    _springapp_jar: "{{ springboot.jar if springboot.jar is defined else _springapp_service_name + '.jar' }}"
```

Comment:
Currently the symlink is called $appname-current.jar. In order to prevent all these old symlinks laying around, is it possible to keep them the same?

Reply:
That one is still created as before:
```
lrwxrwxrwx. 1 root  root         26 Nov 27 08:11 current -> /opt/teams/teams-gui-8.3.8
lrwxrwxrwx. 1 root  root         33 Nov 27 08:12 teams.jar -> /opt/teams/teams-server-8.3.8.jar
```